### ISSUE TITLE
Where are the Perl 5 communities and enthusiasts?

### ISSUE THREAD

**Issue by kytrinyx** [exercism/perl5#203]
As we move towards the launch of the new version of Exercism we are going to be ramping up on actively recruiting people to help provide feedback.

Our goal is to get to 100%: everyone who submits a solution and wants feedback should get feedback. Good feedback. You can read more about this aspect of the new site here: http://mentoring.exercism.io/

To do this, we're going to need a lot more information about where we can find language enthusiasts.

* Is Perl 5 supported by one or more large organizations?
* Does Perl 5 have an official community manager?
* Do you know of specific communities (online or offline) that are enthusiastic about Perl 5? (Chat communities, forums, meetups, student clubs, etc)
* Are there popular conferences for Perl 5? (If so, what are some examples?)
* Are there any organizations who are targeted specifically at getting certain subgroups or demographics interested in Perl 5? (e.g. kids, teenagers, career changers, people belonging to various groups that are typically underrepresented in tech?)
* Are there specific groups or programs dedicated to mentoring people in Perl 5?
* Are there popular newsletters for Perl 5?
* Is Perl 5 taught at programming bootcamps? (If so, what are some examples?)
* Is Perl 5 taught at universities? (If so, what are some examples?)

In other words: where do people care a lot and/or know a lot about Perl 5?

This is part of the project being tracked in https://github.com/exercism/meta/issues/103

**Comment by xdg** [comment]
Some drive-by answers:

* Perl is not supported by a large organization; the official organization in charge of Perl is [The Perl Foundation](https://www.perlfoundation.org/)
* Perl 5 community management is decentralized; several prominent members blog or have newsletters, e.g [The Perl Weekly](http://perlweekly.com/)
* Communities: here is a directory of Perl User Groups internationally: https://www.pm.org/
* Popular conferences:
    * https://perlconference.us/ (which is where I first heard of exercism.io in a lightning talk)
    * http://act.perlconference.org/tpc-2018-glasgow/
    * These and more Perl 5 events: https://www.perl.org/events.html

**Comment by kytrinyx** [comment]
Thank you for these, @xdg!
